src/main/java/ro/vladutit/Don/t/forget/v2/service/CategoryService.java

```diff
@@ -36,4 +36,11 @@ public Category getCategoryById(Long id) {
     public void deleteCategoryById(Long id) {
         categoryRepository.deleteById(id);
     }
+
+    public int getCategoryByName(String name) {
+        Optional<Category> optional = categoryRepository.findByNameIgnoreCase(name);
+        if(optional.isPresent())
+            return 1;
+        return 0;
```

Comment:
either you use brackets for the if, either you return directly optional.isPresent() -> depending on what is returning